metaflow/plugins/kfp/kfp_decorator.py

```diff
@@ -68,6 +70,12 @@ def step_init(self, flow, graph, step, decos, environment, datastore, logger):
         if datastore.TYPE != "s3":
             raise KfpException("The *@kfp* decorator requires --datastore=s3.")
 
+        for deco in decos:
+            if isinstance(deco, AcceleratorDecorator):
+                accelerator_type: str = deco.attributes["type"]
+                if not accelerator_type:
+                    raise Exception("You must specify the type of accelerator.")
```

Comment:
This validation must be in the AcceleratorDecorator not in KfpInternalDecorator

Reply:
And this isn't an `Exception`, it should be a `MetaflowException` like https://github.com/zillow/metaflow/blob/06bfd051cbf4c1657f3175c594ef3ae898a8732f/metaflow/plugins/catch_decorator.py#L10-L10

Reply:
Fixed.